css/properties.json

```diff
@@ -1193,7 +1193,7 @@
     "status": "nonstandard"
   },
   "-webkit-mask": {
-    "syntax": "<mask-image> [ <'-webkit-mask-repeat'> || <'-webkit-mask-attachment'> || <'-webkit-mask-position'> || <'-webkit-mask-origin'> || <'-webkit-mask-clip'> ]*",
+    "syntax": "[ <mask-reference> || <position> [ / <bg-size> ]? || <repeat-style> || [ <geometry-box> | no-clip ] || <composite-style> || <attachment> ]#",
```

Comment:
`-webkit-mask` is not standard and current syntax is more safe. Are you sure a new syntax is relevant to `-webkit-*`  properties?

Reply:
Thanks for your feedback. The reason I noticed it was a bit off was firstly because `<mask-image>` doesn't refer to anything. I get warnings for all missing data types when I generate the typings in csstype. What would you suggest it refers to?

Secondly, several of the `<'-webkit-*'>` in the current syntax accepts comma separated values and my suspicions is that `-webkit-mask` doesn't accept values like `url(...) scroll, fixed, padding, content` like this syntax is telling me that I can. Or have I missed somthing?

Reply:
> The reason I noticed it was a bit off was firstly because `<mask-image>` doesn't refer to anything. I get warnings for all missing data types when I generate the typings in csstype. What would you suggest it refers to?

You're right, `<mask-image>` is not defined in `syntax.json`. As I found I missed this issue, because CSSTree has [a patch where this syntax is defined](https://github.com/csstree/csstree/blob/master/data/patch.json#L636-L639). I suppose we should add such syntaxes to `syntax.json`.

On the other hand, `-webkit-mask-*` properties are not standard and we should mitigate influence of it to syntaxes from a specs. For instance `<mask-image>` can be used by a spec later.

> Secondly, several of the `<'-webkit-*'>` in the current syntax accepts comma separated values and my suspicions is that -webkit-mask doesn't accept values like url(...) scroll, fixed, padding, content like this syntax is telling me that I can. Or have I missed somthing?

Agree, current syntax is not valid by the reasons you're mentioning. So this syntax should be reworked. Unfortunately, it is too hard to found a complete information about `-webkit-mask` properties group syntax. I found an [interesting gist](https://gist.github.com/webtobesocial/a27380c066a03e6c87b2#-webkit-mask) – may be it can help.